Tests/Clappr_Tests/Classes/Plugin/Core/MediaControl/QuickSeekMediaControlPluginTests.swift

```diff
@@ -68,14 +67,64 @@ class QuickSeekMediaControlPluginTests: QuickSpec {
                     }
                 }
                 
-                context("and it colides with another UICorePlugin") {
-                    it("does not seek") {
+                describe("and there is another UICorePlugin") {
+                    beforeEach {
+                        playButton = PlayButton(context: core)
+                        
+                        core.addPlugin(playButton)
+                        core.render()
+                        
+                        mediaControl.render()
                         playButton.view.layoutIfNeeded()
                         mediaControl.view.layoutIfNeeded()
                         
-                        let shouldSeek = quickSeekPlugin.shouldSeek(point: CGPoint(x: 100, y: 100))
+                    }
+                    
+                    context("and it collides with that plugin") {
+                        it("does not seek") {
+                            let playButtonCenterInMediaControlCoordinate = playButton.view.convert(playButton.view.center, to: mediaControl.mediaControlView)
+                            
+                            let shouldSeek = quickSeekPlugin.shouldSeek(point: playButtonCenterInMediaControlCoordinate)
+                            
+                            expect(shouldSeek).to(beFalse())
+                        }
+                    }
+                    
+                    context("and it does not collide with that plugin") {
+                        it("does seek") {
+                            let pointOutsidePlayButton = CGPoint(x: playButton.view.frame.width + 1, y: playButton.view.frame.height + 1)
+                            let outsidePointInMediaControlCoordinate = playButton.view.convert(pointOutsidePlayButton, to: mediaControl.mediaControlView)
+                            
+                            let shouldSeek = quickSeekPlugin.shouldSeek(point: outsidePointInMediaControlCoordinate)
+                            
+                            expect(shouldSeek).to(beTrue())
+                        }
+                    }
+                    
+                    context("and that plugin is not visible") {
+                        it("does seek") {
+                            let playButtonCenterInMediaControlCoordinate = playButton.view.convert(playButton.view.center, to: mediaControl.mediaControlView)
+                            playButton.view.alpha = 0.0
+                            
+                            let shouldSeek = quickSeekPlugin.shouldSeek(point: playButtonCenterInMediaControlCoordinate)
+                            
+                            expect(shouldSeek).to(beTrue())
+                        }
+                    }
+                }
+                
+                context("and there are overlay plugins") {
+                    it("ignores them and seeks") {
```

Comment:
This phrase makes sense? If we have an overlay plugin like the config screen it should seek?

Reply:
Actually it should not seek, but we do have another way of handling that.
QuickSeekPlugin listens to the `didShowOverlayPlugin` and `didHideOverlayPlugin` events, and when they're triggered it removes and adds the gesture, respectively.

Reply:
We have a different way of handling `Overlay Plugins`, by listening to some show and hide events triggered by them, so it should not be necessary to take that in consideration here. We only added that test because we must ignore those kind of plugins in the `filteredOutPlugins` method, knowing that regardless of being visible or not, they still exist in the plugins list.

Reply:
Maybe an example can make things clearer:
- If there is a visible OverlayPlugin, then we won't seek because we won't have a gesture recognizer to capture the taps. If there isn't a visible one, then we should ignore them in the filtering and seek.

Reply:
The context look like the opposite to me:
- If there is a visible OverlayPlugin then we should ignore it in the filtering and seek.

Make sense?